src/main/java/org/opentripplanner/netex/mapping/StopTimesMapper.java

```diff
@@ -128,14 +128,16 @@ StopTimesMapperResult mapToStopTimes(
 
       StopPointInJourneyPattern stopPoint = findStopPoint(pointInJourneyPattern, journeyPattern);
 
+      StopLocation stop = lookUpStopLocation(stopPoint);
       if (
         stopPoint != null &&
-        (isFalse(stopPoint.isForAlighting()) && isFalse(stopPoint.isForBoarding()))
+        stop == null &&
+        isFalse(stopPoint.isForAlighting()) &&
+        isFalse(stopPoint.isForBoarding())
       ) {
         continue;
       }
 
-      StopLocation stop = lookUpStopLocation(stopPoint);
       if (stop == null) {
```

Comment:
What if you just add the `isFalse(stopPoint.isForAlighting()) && isFalse(stopPoint.isForBoarding())` inside this if-statement and continue if that is true, otherwise log the issue and return? `lookUpStopLocation` creates already an issue in that case?

Reply:
That's a good point, we overlooked that there is already a check for `stop == null`. But I think we still need the check for `stopPoint != null`.

Reply:
That is done already inside `lookUpStopLocation`